styles.css

```diff
@@ -42,10 +42,10 @@ h1 {
 
 .selected {
   font-size: 2em;
-  -webkit-animation: highlight-selected 1.5s infinite; /* Safari 4+ */
-  -moz-animation: highlight-selected 1.5s infinite; /* Fx 5+ */
-  -o-animation: highlight-selected 1.5s infinite; /* Opera 12+ */
-  animation: highlight-selected 1.5s infinite; /* IE 10+, Fx 29+ */
+  -webkit-animation: highlight-selected 1s infinite; /* Safari 4+ */
```

Comment:
In general, you can omit CSS vendor prefixes unless you know for sure that you need to support finicky browsers.
https://caniuse.com/#search=animation
I recommend just sticking with `animation` here.